tests/test_freetextsearch.py

```diff
@@ -51,7 +1364,6 @@ def testSearchFields(self):
         self.assertTrue(response.status_code == 200)
         json_response = json.loads(response.data.decode('utf-8'))
         self.assertEqual(len(json_response['data']), 1)
```

Comment:
my original idea (miscomunicated) was to just return highlights for the fields that were requested, but not for others. But now I can see it is a legitimate case to ask for 2 fields, but have the full highlight back in case the UI needs more details on what was matched.
Ideally we should be able to disable the highlights with a separate option, but don't put them in the 'fields' options that are applied to filter the source. @emepyc @rgasparyan  do you believe it is more intuitive as a user to filter the highlights in the fields object?